src/Packetery/Module/Order/PacketCanceller.php

```diff
@@ -164,23 +164,17 @@ public function cancelPacket( Entity\Order $order ): void {
 		$request = new Soap\Request\CancelPacket( (int) $order->getPacketId() );
 		$result  = $this->soapApiClient->cancelPacket( $request );
 
+		$record         = new Log\Record();
```

Comment:
Tento refactor není třeba,  ticket to nevyžaduje.  Prosím uvést do původního stavu.